kvmagent/kvmagent/plugins/host_plugin.py

```
@@ -3666,6 +3668,90 @@ def detach_volume__path(self, req):
+            reporter.progress_report(int(percent))
+
+        def use_wget(url, name, workdir, timeout):
+            return linux.wget(url, workdir=workdir, rename=name, timeout=timeout, interval=2, callback=percentage_callback, callback_data=url)
+
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
+
+        t_shell = traceable_shell.get_shell(cmd)
+        rsp = DownloadResponse()
+
+
+        # 检测物理机 指定目录的容量，如果容量不足 则报错返回。
```

Ruff (0.13.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)